### ISSUE TITLE
Different sub-channels

### ISSUE THREAD

**Issue by AstudilloMarillo** [bmlzootown/Hydravion-AndroidTV#31]
Hi bmlzootown, I'm so happy that you made this application, I just wish I found it earlier so I wouldn't have had to struggle with the less-than-optimal chromecast feature of the iOS app of Floatplane for as long as I did :p.

I was wondering if it were possible to add the different sub-channel categories that the Floatplane website now offers. So for example for LTT it's Linus Tech Tips, ShortCircuit, Mac Address, FP Exclusives etc. Right now, it's not really easy to identify what sub-channel each video is from. 

An easier way (I think) could maybe be to just add the icons of the channels in front of the title of the videos, just like the Floatplane website has. That way, at least, it'd be easy to distinguish the different sub-channels.

**Comment by insaneFu** [comment]
agreed. I would also like a feature like this very much. maybe even a filter for those subchannels, so I can more easily (or at all even, if browsed unregularly) get directly to the floatplane exclusive goodness 💟
